tst/testinstall/list.tst

```diff
@@ -309,6 +309,29 @@ gap> l;
 gap> TNAM_OBJ(l);
 "plain list (rectangular table)"
 
+# Check PlainListCopy
+gap> checkPlainListCopy := function(l)
+>   local copy, tnum;
+>   copy := PlainListCopy(l);
+>   tnum := TNUM_OBJ(l);
+>   return IsPlistRep(copy) and l = copy and
+>          not IsIdenticalObj(l,copy) and TNUM_OBJ(l) = tnum;
```

Comment:
I don't get the purpose of the `TNUM_OBJ(l) = tnum` here, since you've just defined `tnum := TNUM_OBJ(l);` two lines above.

Is it to make sure that the `TNUM` of `l` didn't change while computing `l = copy` and `IsIdenticalObj(l,copy)`? Because they're the only things involving `l` that happen in between.

Reply:
That is a mistake, I wanted to check PlainListCopy didn't change anything about l, and put the TNUM_OBJ in the wrong place.